supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -70,6 +70,7 @@ object PipelinesApiAsyncBackendJobExecutionActor {
 
   // If the JES code is 2 (UNKNOWN), this sub-string indicates preemption:
   val FailedToStartDueToPreemptionSubstring = "failed to start due to preemption"
+  val FailedV2Style = "The assigned worker has failed to complete the operation"
```

Comment:
To make it consistent with line 72, how about `FailedToCompleteTheOperationSubstring`?